### PR TITLE
security overview 

### PR DESCRIPTION
**Pull request by fiquick**
Adding an intro section to the existing security docs

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-489.surge.sh

Updated pages:
- [Security architecture](https://neo4j-docs-aura-489.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-489.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-489.surge.sh/aura/platform/security/secure-connections)

**Comment by fiquick** [comment]
[Aura security - architecture diagram](https://trello.com/c/jq5pfg48/310-aura-security-architecture-diagram)

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 1]
```diff
@@ -1,17 +1,37 @@
...
+:description: Secure, isolated infrastructure with fine-grained access control, encryption, compliance, and key security features like SSO and Customer Managed Encryption keys.
+
+label:AuraDB-Virtual-Dedicated-Cloud[] 
+label:AuraDS-Enterprise[]
+
+== Introduction to the security architecture
+
+100% of AuraDB Virtual Dedicated Cloud customers operate within a Virtual Private Cloud (VPC) isolated architecture with a dedicated infrastructure.
+
+AuraDB offers fine-grained access control to the database with built-in Role-Based Access Control (RBAC) at the database level, similar to what can be configured in a self-managed Neo4j environment.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:12]: Is this described anywhere?

**Review by AlexicaWright — COMMENTED** [review]
I think we need to rethink this section. It is useful to give an introduction to the security architecture in AuraDB, but it needs to be a) its own subchapter as it introduces not only secure connections but alos SSO and encryption, and b) more concise and less repetitive.
Also, we don't refer to users as customers in the docs and we are not trying to sell anything.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-489.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-489.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-489.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-489.surge.sh/aura/platform/security/secure-connections)

**Comment by fiquick** [comment]
Closing and putting into a word doc - intro to security

**Pull request closed without merging**